source/views/settings/sections/odds-and-ends.js

```diff
@@ -29,11 +43,24 @@ class OddsAndEndsSection extends React.Component {
       url: 'https://github.com/StoDevX/AAO-React-Native',
       id: 'ContributingView',
     })
+  onAppIconButton = () => this.onPressButton('IconSettingsView')
 
   render() {
+    if (this.state.loading) {
+      return <View />
+    }
```

Comment:
Remove because of next comment

Reply:
I didn’t like this to begin with. Thanks for a better idea.